src/index.tsx

```diff
@@ -1,13 +1,32 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { HomePage } from './components/HomePage';
+import { PeoplePage } from './components/PeoplePage';
 
 createRoot(document.getElementById('root') as HTMLDivElement).render(
   <Router>
-    <App />
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+
+        <Route path="people">
+          <Route path=":personId?" element={<PeoplePage />} />
+        </Route>
+
+        <Route path="*" element={<h1 className="title">Page not found</h1>} />
+
+        <Route path="home" element={<Navigate to=".." replace />} />
```

Comment:
The `Navigate` component is used to redirect from `/home` to `..` (the parent route). Ensure that this redirection logic is correct and aligns with the intended navigation flow of the application.